src/App.css

```diff
@@ -6,18 +6,55 @@
   animation: App-logo-spin infinite 20s linear;
   height: 80px;
 }
-
+.ToDoItemInput{
```

Comment:
We usually style class like `.todo-item-input` without capital and with dash separators.